.github/workflows/integration_test.yaml

```diff
@@ -94,12 +94,13 @@ on:
         type: string
         description: Target directory to do the Trivy testing
         default: "."
+      trivy-image-enabled:
```

Comment:
Maybe we can run it always? Is there any reason to skip it? It is my understanding that we can provide config to ignore issues

Reply:
Hm...Since all the other tests has the option to enable/disable, I left this one too. In case you want to disable everything just do the integration test.

Reply:
I was thinking that it would make forcing compliance easier if we don't allow disabling it. Same approach as the linters

Reply:
You convinced me :)